pkg/scheduler/framework/plugins/defaultassigner/dynamic_test.go

```diff
@@ -297,15 +297,15 @@ func TestDynamicAssigner_Assign(t *testing.T) {
 
 			status := pl.PreAssign(context.TODO(), state, tt.args.sub, tt.args.finv, tt.args.availableReplicas)
 			if status != nil {
-				t.Errorf("PreAssign() got status = %v, want nil", status)
+				t.Errorf("\n%s PreAssign()\ngot status = %v,\nwant nil", tt.name, status)
```

Comment:
Will `\n` make the output hard to read?

Reply:
`\n` easy to  compare want and result when run file test, but useless with run `go test `, I rollback that